themes/theme-unicorn/src/colors.ts

```diff
@@ -0,0 +1,36 @@
+export const colors = {
```

Comment:
why the file here?

Reply:
I thought it's good to have the colors in unicorn theme.